graphql/README.md

```diff
@@ -0,0 +1,74 @@
+## GraphQL Developer Guide
+
+### Modifying the GraphQL Schema
+
+#### Fields
+
+You can add fields to the .graphql files in the `/schema` folder. When you run `make gqlgen`, these changes will be processed and a resolver will be generated for you.
```

Comment:
Since the directory is relative to `graphq/`
```suggestion
Add fields to the .graphql files in the `schema/` folder. When you run `make gqlgen`, these changes will be processed and a resolver will be generated.
```